README-sc20.txt

```diff
@@ -0,0 +1,44 @@
+In the SC20 list, only the new C-application benchmark will be used.
+
+The io500.sh script can optionally be used to prepare the output
```

Comment:
I personally do not like artificial line breaks but have editors handle this as this allows everyone to enjoy line breaks at the width of their choice.
Also IMHO improves GIT merges to separate by sentence. But surely I'm OK with the choice you made. Just for discussion.

Reply:
Yeah, when I used to program a lot, I got used to 80 char wide lines.  That's my typical preference.